tools/pyprophet/pyprophet_export.xml

```diff
@@ -0,0 +1,372 @@
+<tool id="gp_pyprophet_export" name="PyProphet_export" version="@VERSION@.0">
```

Comment:
please change all IDs to pyprophet_ ... and in the name I would remove the underscore.